### ISSUE TITLE
Email addresses are shared with other users

### ISSUE THREAD

**Issue by brettclanton001** [brettclanton001/Dashous#2]
As a Dashous user
When I approve an offer
I want the other party to receive a generated email address
and emails to that email address will forward to my email address
Because I want the communication with this other user to be limited to this trade
and I don't want them to have my personal email address

**Comment by Jzarecta** [comment]
I dont understand that well what you meant, you want email alerts? or you want the site to build custom email address for users? I think communication should be handled on the site. 
Also will debate how open these negotiations comes in order to 'settle disputes' and/or have custodians or arbitrators in the future. Check how Bitsquare handles this:
https://www.youtube.com/watch?v=MVUhPh76vOQ

**Comment by brettclanton001** [comment]
Yeah, at the time I was thinking of generated email addresses.  They would direct to my server and I would forward to the real email address.

Right now, I think I agree that the communication should be handled within the site.  The issue is that I don't actually want to be able to view all of this communication.  So I'm working on implementing an encrypted chat to eliminate the need for emails.

An arbitration service within Dashous will be a conversation for a little bit later. :)
